pulsar-io/hbase/src/main/java/org/apache/pulsar/io/hbase/HbaseAbstractConfig.java

```diff
@@ -38,12 +36,9 @@
 @EqualsAndHashCode
 @ToString
 @Accessors(chain = true)
-public class HbaseAbstractConfig implements Serializable {
```

Comment:
better to keep  Serializable

Reply:
think AbstractConfig not ser, extend class imp ser.

Reply:
done.